governance/src/utils.js

```diff
@@ -0,0 +1,55 @@
+function getAbi(abiName) {
+  // eslint-disable-next-line global-require,import/no-dynamic-require
+  const { abi } = require(`../abi/${abiName}`);
+  return abi;
+}
+
+function createProposalFromLog(log) {
+  const proposalId = log.args.proposalId.toString();
+  const proposal = {
+    proposalId,
+    proposer: log.args.proposer,
+    targets: log.args.targets.join(','),
+    _values: (log.args[3].map((v) => v.toString())).join(','),
```

Comment:
Add a comment explaining why this argument has to be parsed from the log differently